packages/manager/apps/hycu/src/pages/onboarding/Onboarding.page.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Card, OnboardingLayout } from '@ovh-ux/manager-react-components';
+import useGuideUtils from '@/hooks/guide/useGuideUtils';
+import Breadcrumb from '@/components/Breadcrumb/Breadcrumb';
+import onboardingImgSrc from './onboarding-img.png';
+
+export default function Onboarding() {
+  const { t } = useTranslation('onboarding');
+  const link = useGuideUtils();
+
+  const tileList = [
+    {
+      id: 1,
+      texts: {
+        title: t('guide1Title'),
+        description: t('guide1Description'),
+        category: t('guideCategory'),
+      },
+      href: link?.guideLink1,
+    },
+    {
+      id: 2,
+      texts: {
+        title: t('guide2Title'),
+        description: t('guide2Description'),
+        category: t('guideCategory'),
+      },
+      href: link?.guideLink2,
+    },
+    {
+      id: 3,
+      texts: {
+        title: t('guide3Title'),
+        description: t('guide3Description'),
+        category: t('guideCategory'),
+      },
+      href: link?.guideLink3,
+    },
+  ];
```

Comment:
```suggestion
  const TILE_LIST = [
    {
      id: 1,
      texts: {
        title: t('guide1Title'),
        description: t('guide1Description'),
        category: t('guideCategory'),
      },
      href: link?.guideLink1,
    },
    {
      id: 2,
      texts: {
        title: t('guide2Title'),
        description: t('guide2Description'),
        category: t('guideCategory'),
      },
      href: link?.guideLink2,
    },
    {
      id: 3,
      texts: {
        title: t('guide3Title'),
        description: t('guide3Description'),
        category: t('guideCategory'),
      },
      href: link?.guideLink3,
    },
  ] as const;
```

Reply:
The onboarding page will be updated so I ignore it for now